deep-learning/_posts/2017-03-16-resnet.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: review
+title: (ResNet) Deep Residual Learning for Image Recognition
+tags: deep-learning CNN essentials classification
+cite:
+    authors: "K He, X Zhang, S. Ren, J. Sun"
+    title:   "Deep Residual Learning for Image Recognition"
+    venue:   "Proceedings of CVPR 2016, p.770-778"
+pdf:   "http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf"
+---
+
+ 
+Winner in several tracks of the ILSVRC & COCO 2015 competitions: ImageNet detection, ImageNet localization, COCO detection, and COCO segmentation.  The main novelty of this paper is the use of residual blocks which is a series of 2 or 3 conv layers with a skip connection.
+ 
+![](/deep-learning/images/resnet/resnet2.jpg)
+
+The skip connection works as identity shortcuts to remove the unuseful blocks and thus to build much deeper architectures
```

Comment:
I suggest those changes:

"The skip connection works as an identity shortcut to skip a block when it is not useful for a particular class. This allows to build much deeper architectures."

When a block is skipped, it's not completely useless. If it was the case, we would just remove it.